authentication/admin.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib import admin
```

Comment:
I've created classes for each kind of user that can login to the application. Whenever a user of this type is created its correct permissions are assigned.